spark/dl/src/main/scala/com/intel/analytics/bigdl/nn/RReLU.scala

```diff
@@ -46,8 +46,8 @@ import scala.reflect.ClassTag
  */
 @SerialVersionUID(- 9012115082607155821L)
 class RReLU[T: ClassTag](
-  lower: Double = 1.0/8,
-  upper: Double = 1.0/3,
+  val lower: Double = 1.0/8,
```

Comment:
seems this one can be optimized by compiler and cannot be accessed from constructor thus leave it as it is